packages/react-vapor/src/components/slider/Slider.tsx

```diff
@@ -24,11 +24,11 @@ export enum AppendedValueSide {
     both = 'BOTH',
 }
 
-export interface SliderOwnProps extends CommonApiProps {
```

Comment:
Why we were using this before? (Genuinely curious).

Reply:
The react vapor slider leverage the rc-slider and we must leverage the lib types.  I dont know the difference between the CommonApi and the SliderProps though

Reply:
CommonApi has no `onAfterChange` prop, which you used in your ab test slider @dmgauthier, but SliderProps does (and extends CommonApi)